Makefile

```diff
@@ -44,6 +44,8 @@ CATCH2_URL := https://github.com/catchorg/Catch2.git
 CATCH2_VER := v2.13.6
 LCOV_URL := https://github.com/linux-test-project/lcov.git
 LCOV_VER := v2.0
+DEP_CC := gcc
+DEP_CXX := g++
```

Comment:
If these are used only for third party dependencies, add a comment here to explain that